stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/persistance/OfflineSupport.kt

```diff
@@ -0,0 +1,17 @@
+package io.getstream.chat.android.offline.experimental.persistance
+
+import io.getstream.chat.android.core.ExperimentalStreamChatApi
+import io.getstream.chat.android.offline.experimental.global.GlobalState
+import io.getstream.chat.android.offline.experimental.plugin.OfflinePluginImpl
+import io.getstream.chat.android.offline.experimental.plugin.logic.LogicRegistry
+import io.getstream.chat.android.offline.experimental.plugin.state.StateRegistry
+
+/**
+ * The entry point of all offline state ([OfflinePluginImpl.state]) and behavior ([OfflinePluginImpl.logic]).
+ */
+@ExperimentalStreamChatApi
+internal class OfflineSupport(
```

Comment:
Do we need this class? I think we have other ways of accessing different States/Logic

Reply:
Yep. That's just noise from interaction with the code. I should have deleted.